internal/app/agent/service.go

```diff
@@ -20,11 +22,16 @@ import (
 	"github.com/nats-io/go-nats"
 )
 
+type cmdType string
```

Comment:
Do we need this?

Reply:
I like it and I think that @dusanb94 suggested same on some other place